libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -60,6 +60,11 @@ class AP_ICEngine {
 
     enum ICE_State state;
 
+    // filter for RPM value
+    float rpm_value;
```

Comment:
I don't think this needs to be here?

Reply:
I moved it from a scoped local in update() to an object property. I moved it to a higher scope in update().